osdconfig/api_test.go

```diff
@@ -75,6 +75,17 @@ func TestSetGetNode(t *testing.T) {
 	if !reflect.DeepEqual(expectedConf, receivedConf) {
 		t.Fatal("expected and received values are not deep equal")
 	}
+
+	// now delete the node
+	if err := manager.DelNodeConf(expectedConf.NodeId); err != nil {
+		t.Fatal("error in deleting node config")
```

Comment:
I see this code uses `t.Fatal()` a lot, but instead start using `testify.assert(...)`